src/gui/drivepreferenceswidget.cpp

```diff
@@ -1042,17 +1057,30 @@ void DrivePreferencesWidget::onErrorAdded() {
 }
 
 void DrivePreferencesWidget::onRemoveDrive(bool checked) {
-    Q_UNUSED(checked)
+    Q_UNUSED(checked);
```

Comment:
Q_UNUSED macro already contains the ";" ;)
#define Q_UNUSED(x) (void)x;

Reply:
Thanks, done.